docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md

```diff
@@ -0,0 +1,97 @@
+---
+type: docs
+title: "How-To: Migrate from existing Kubernetes resources"
+linkTitle: "Migrate from YAML"
+description: "Learn how to migrate to a Radius container from existing Kubernetes YAML"
+weight: 500
+categories: "How-To"
+tags: ["containers","Kubernetes"]
+---
+
+This guide will provide an overview of how to migrate your existing Kubernetes resources from [Kubernetes YAML](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec).
+
+### Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+### Step 1: Define a Radius application and container
+
+Define a Radius Application resource that will contain your Kubernetes resource container.
+
+> Radius resources, when deployed to a Kubernetes environment, are mapped to one or more Kubernetes objects. For more information, visit [Radius on Kubernetes platform]({{< ref "guides/operations/kubernetes#resource-mapping" >}}).
+
+{{< rad file="snippets/basemanifest.bicep" embed=true marker="//APPLICATION" >}}
+
+### Step 2: Define a base Kubernetes YAML file
+
+In a file named `kubernetes.yaml` define all the Kubernetes resources that correspond to the new Radius container.
+
+Note that the names of the Deployments, Services, and Config Maps all must match each other and the name of the new Radius container. Any other resources can be named differently.
+
+{{< rad file="snippets/basemanifest.yaml" embed=true lang="yaml">}}
+
+### Step 3: Add the Kubernetes base YAML configurations to your Radius application definition
+
+
+#### Container configurations
+
+Define your Radius container resource port with the port of your Kubernetes `Service` as well as the image container location that your service is dependent on.
+
+#### Runtime configuration
+
+Load your Kubernetes YAML file into your Radius container resource through the `properties.runtimes.kubernetes.base` property which expects a string value containing all your Kubernetes data.
```

Comment:
Instead of H4 headers I'd make this an ordered list of steps. (1. Define your.... 1. Load your Kubernetes.....)